internal/cluster/transition.go

```diff
@@ -99,12 +99,15 @@ func (th *transitionHandler) PostPrepareForInstallation(sw stateswitch.StateSwit
 		return errors.New("PostPrepareForInstallation invalid argument")
 	}
 
-	if err := params.ntpUtils.AddChronyManifest(params.ctx, logutil.FromContext(params.ctx, th.log), sCluster.cluster); err != nil {
+	if err := params.manifestsGenerator.AddChronyManifest(params.ctx, logutil.FromContext(params.ctx, th.log), sCluster.cluster); err != nil {
 		return errors.Wrap(err, "PostPrepareForInstallation failed to add chrony manifest")
 	}
 
 	sendNTPMetric(logutil.FromContext(params.ctx, th.log), params.metricApi, sCluster.cluster)
 
+	if err := params.manifestsGenerator.AddIpv6Manifest(params.ctx, logutil.FromContext(params.ctx, th.log), sCluster.cluster); err != nil {
```

Comment:
Shouldn't we call this only if using IPv6?

Reply:
The manifest adds settings for IPv6 only so it will have no effect on IPv4